.circleci/config.yml

```diff
@@ -170,22 +194,36 @@ workflows:
           filters:
             tags:
               only: /.*/
-      - docker-build:
+      - setup-env:
           requires:
             - frontend-services-checks
             - backend-services-checks
           filters:
             tags:
               only: /.*/
+      - docker-build-backend:
+          requires:
+            - setup-env
+          filters:
+            tags:
+              only: /.*/
+      - docker-build-frontend:
+          requires:
+            - setup-env
```

Comment:
https://github.com/litmuschaos/litmus/pull/2036/files#r487068594

Reply:
Same as the above